source/includes/_endpoint-deck.md

```diff
@@ -0,0 +1,413 @@
+    ]
+}
+```
+
+#### PATCH
+
+To make changes to the order, a client should PATCH the full `shoji:order`
+resource to the endpoint with the new order on its `graph` attribute.
+
+Any slide not mentioned on the payload will be added at the end of the graph
+in arbitrary order.
+
+
+```json
+{
+    "element": "shoji:order",
+    "self": "/api/datasets/123/decks/123/slides/flat/",
+    "description": "Order of the slides on this deck",
+    "graph": [
+        "/api/datasets/123/decks/123/slides/123/",
+        "/api/datasets/123/decks/123/slides/456/"
+    ]
+}
+```
+
+This is a flat order: grouping or nesting is not allowed.
+
+## Analysis
+
+Each slide contains one analysis. An analysis -- a table or graph with some specific combination of variables
+defining measures, rows, columns, and tabs; settings such as percentage
+direction and decimal places -- can be saved to a _deck_, which can then be
+exported, or the analysis can be reloaded in whole in the application or even
+exported as a standalone embeddable result.
+
+An analysis is defined by a _query_, _query environment_, and _display settings_.
+To save an analysis, `POST` these to a deck.
+
+<aside class="notice">
+Analysis queries are described in detail in the
+[feature guide](#multidimensional-analysis). [Filters](#filters) may contain a
+mix of stored filters or expressions. Expressions may contain an optional
+`name` which may be used to label results.
+</aside>
+
+Display settings can be anything a client may need to reproduce the view of the
+data returned from the query. The settings the Crunch web client uses are shown
+here, but other clients are free to store other attributes as they see fit.
+Display settings should be objects with a `value` member.
+
+
+```json
+{
+    "query": {
+        "dimensions" : [],
+        "measures": {}
+    },
+    "query_environment": {
+        "filter": [
+            {"filter": "<url>"},
+            {"function": "expression", "args": [], "name": "(Optional)"}
+        ],
+        "weight": "url"
+    },
+    "display_settings": {
+        "decimalPlaces": {
+            "value": 0
+        },
+        "percentageDirection": {
+            "value": "colPct"
+        },
+        "vizType": {
+            "value": "table"
+        },
+        "countsOrPercents": {
+            "value": "percent"
+        },
+        "uiView": {
+        	"value": "expanded"
+	    }
+    }
+}
+```
+
+Name | Description
+-----|-------------
+query | Includes the query body for this analysis
+query_environment | An object with a `weight` and `filters` to be used for rendering/evaluating this analysis
+display_settings | An object containing client specific instructions on how to recreate the analysis
+
+Analyses cannot be edited. To make changes, you can load the analysis,
+make modifications to it and POST again to the deck's slides catalog to create
+a new slide with the updated analysis and DELETE the old one, if desired.
```

Comment:
I get that whaam doesn't edit analyses/slides, but it seems odd from an API perspective that we wouldn't have a PATCH method.